package-testing/android-relay/app/src/main/java/cloud/eppo/android/sdkrelay/SocketHandler.kt

```diff
@@ -0,0 +1,32 @@
+package cloud.eppo.android.sdkrelay
+
+import io.socket.client.IO
+import io.socket.client.Socket
+import java.net.URISyntaxException
+
+object SocketHandler {
+
+  lateinit var mSocket: Socket
```

Comment:
kotlin seems pretty sweet

Reply:
It's like java - headache + typescript + something-else-that-makes-it-all-more-elegant

Reply:
AFAIK hungarian notation is not used in kotlin, just call it `socket`